.ci/integration_test

```diff
@@ -134,6 +134,11 @@ function create_etcd_data_directory() {
   export ETCD_DATA_DIR=${TEST_DIR}/etcd-data
   mkdir -p ${ETCD_DATA_DIR}
 }
+
+function get_aws_existing_region() {
+  export REGION=`cat ${HOME}/.aws/config | grep -e "^.*region.*$" | awk '{print $3}'`
```

Comment:
Concerning the AWS documentation https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html the default config file looks like the following:
```
[default]
region=us-west-2
output=json
```

There are not white spaces and thus  `awk '{print $3}'` won't work. If you want to cover both options then you can normalize the field before extracting the region information.

Reply:
>  There are no white spaces and thus awk '{print $3}' won't work

ok, but in my local `~/.aws/config` file have spaces and it is working fine.
```
[default]
region = eu-west-1
```
but yeah thanks .... I will try to generalise this.

Reply:
Please see https://github.com/gardener/etcd-backup-restore/pull/496/files#r947653005: We should try to solve the issue for the community and thus appreciate the generalization you offered https://github.com/gardener/etcd-backup-restore/pull/496/files#r949200282. That should only be a small change.

Reply:
done. PTAL